### PR TITLE
[Chore][#124] Prepare new release and reactivate tests

### PR DESCRIPTION
**Pull request by pasqu4le**
## Description

This PR includes some doc improvement and a version bump for a new release of the contract.

Additionally it reactivates network tests, as it's now possible to run them in ~10 mins, which is not too bad.
Note: it doesn't close the issue because they could be faster with batching, which was recently merged in `morley`, but requires an update of `morley-ledgers` to be used here.

## Related issue(s)

Relates part of #124

#### Related changes (conditional)

- Tests
  - [x] If I added new functionality, I added tests covering it.
  - [x] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

[//]: # (Add more docs here if you have them in the repository)
- Documentation
  - [x] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)
    - Haddock

#### Stylistic guide (mandatory)

- [x] My commits comply with [the following policy](https://www.notion.so/serokell/Commit-and-PR-policy-4cf98e1a910a415d86b5f2491d9af1af).
- [x] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

**Comment by pasqu4le** [comment]
Note for reviewers: the README(s) mention the latest release, however the one matching this PR is the [v0.2 pre-release](https://github.com/tqtezos/baseDAO/releases/tag/v0.2-alpha) that will be made official once this gets merged.

File: README.md [hunk 1]
```diff
@@ -5,15 +5,7 @@ BaseDAO is a generic smart contract on Tezos that enables a community to collect
...
 
-This repository provides:
-* Generic DAO contract, written in cameLIGO, that can be used for various DAOs implementations.
-* Instantiations of this contract:
-  1. Simplest implementation with no custom logic ([TrivialDAO](#trivialdao))
-  2. Two more practical DAO contracts: [RegistryDAO](#registrydao) and [TreasuryDAO](#treasurydao).
-* A [TypeScript API](#typescript-api) to interact with DAO contracts.
-* [Contract tests](#testing) to validate its implementation.
-
-## Contract documentation and configurations
```
Comment (sras) [inline README.md:16]: This section appear to be removed, and does not appear to be included anywhere else. Was that intentional?

File: README.md [hunk 2]
```diff
@@ -22,172 +14,55 @@ contract, meaning that DAOs with different logic and needs can be obtained by
...
-
-As for the other examples, all the arguments are optional and will be equal to
-the values above if not specified.
+
+## Deploying/Originating
+
+This contract is too large to fit inside the current Tezos origination limit.
+
+To overcome this limitation we advice to use the [morley-large-originator](https://gitlab.com/morley-framework/morley/-/tree/master/code/morley-large-originator)
+tool or the workaround that it's described there.
```
Comment (sras) [inline README.md:49]: > that it's described

"that it's" seems a bit weird.

File: docs/building.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+# Building BaseDAO
+
+BaseDAO is implemented using the [LIGO programming language](https://ligolang.org/),
+in particular its CameLIGO dialect.
+
+To generate the contract code, or the storage for one of the provided examples,
+you will need the [ligo executable](https://ligolang.org/docs/intro/installation) installed.
+
+The latest working version tested is [0.13.0](https://gitlab.com/ligolang/ligo/-/releases/0.13.0).
```
Comment (sras) [inline docs/building.md:14]: AFAIU this this version will cause a test failure when running the tests locally. I understand that is this is probably a Morley issue, AFAIU but if someone clones the repo and run the tests, they ll get a test failure. Should we do something about it for this release?

File: README.md (hunk 1 above)
Comment (pasqu4le) [reply README.md:16]: Yep, that was intentional.

That section was a quick summary with more details below, but now that the README's details have been moved to other doc files this is no longer necessary.

My idea is that now the README itself is the summary.

File: docs/building.md (hunk 3 above)
Comment (pasqu4le) [reply docs/building.md:14]: Good point, I forgot about it.

The latest `morley` should indeed solve this, but as I wrote in the description `morley-ledgers` needs to be updated for that to be used here.

It looks like the update should not take much, I'll try to do that and try to update the Ligo used in CI to match this version as well.

Comment (pasqu4le) [reply docs/building.md]: @sras I updated `morley` to the latest release, but that doesn't contain the necessary changes yet (they are on master, but `morley-ledger` once again needs an update).

So I resorted to downgrade the ligo version used to `0.10.0` for now and updated CI accordingly.

Comment (sras) [reply docs/building.md]: It works now. :+1:

**Review by sras — APPROVED** [review]

**Pull request merged**